proto/api.proto

```diff
@@ -51,6 +51,7 @@ message Batch {
 message TableID {
     uint64 db_id  = 1; 
     uint64 tb_id  = 2;
+    uint64 partition_id = 3;
```

Comment:
What is the meaning of parition id? dn did the partition? How is it divided?

Reply:
Requested by @fengttt, `partition_id` would be used in the future.
Just ignore it for now.

Reply:
I would update the design doc to keep up with the new `proto` file.